packages/edit-site/src/components/left-sidebar/navigation-panel/constants.js

```diff
@@ -1,53 +1,51 @@
+/**
+ * WordPress dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+
 export const TEMPLATES_DEFAULT_DETAILS = {
 	// General
 	'front-page': {
-		title: 'Front page',
-		description: '',
+		title: _x( 'Front Page', 'template name' ),
+		description: __(
+			'Front page template, for both a static page or blog posts index'
+		),
 	},
 	archive: {
-		title: 'Archive',
-		description:
-			'Displays the content lists when no other template is found',
-	},
-	single: {
-		title: 'Single',
-		description: 'Displays the content of a single post',
+		title: _x( 'Archive', 'template name' ),
+		description: __( 'Generic archive template' ),
 	},
 	singular: {
-		title: 'Singular',
-		description: 'Displays the content of a single page',
+		title: _x( 'Singular', 'template name' ),
+		description: __( 'Default template for both single posts and pages' ),
 	},
 	index: {
-		title: 'Default (index)',
-		description: 'Displays the content of a single page',
+		title: _x( 'Index', 'template name' ),
+		description: __( 'Default template' ),
 	},
 	search: {
-		title: 'Search results',
-		description: '',
+		title: _x( 'Search Results', 'template name' ),
+		description: __( 'Search results template' ),
 	},
 	'404': {
-		title: '404',
-		description: 'Displayed when a non-existing page requested',
+		title: _x( '404 (Not Found)', 'template name' ),
+		description: __( 'Template for "not found" errors' ),
 	},
 
 	// Pages
 	page: {
-		title: 'Default (Page)',
-		description: 'Displays the content of a single page',
+		title: __( 'Page' ),
+		description: __( 'Single page template' ),
 	},
 
 	// Posts
 	home: {
-		title: 'Posts (home)',
-		description: 'Displayed on your homepage',
+		title: __( 'Home Page' ),
+		description: __( 'Template for the latest blog posts' ),
 	},
-	'archive-post': {
-		title: 'Default (Post archive)',
-		description: 'Displays a list of posts',
-	},
-	'single-post': {
-		title: 'Default (Single post)',
-		description: 'Displays the content of a single post',
+	single: {
+		title: _x( 'Single', 'template name' ),
+		description: __( 'Single post template' ),
```

Comment:
You moved "single" under "Posts" rather than "General". And removed "archive-post" and "single-post".

What's the reason behind this? `single` template will be used for any custom post type. `single-post` is specific to post post type. In my head "Posts" menu refers to `post` posts, you know blog posts and not just generic `wp_post`. (now that's confusing 😹 )

I mean, it doesn't matter at all right now, I'm just curious 😄

Reply:
The reason for removing `archive-post` and `single-post` is that they are not "general" templates for posts, but are specific to a post type (e.g. `archive-${ postType }`).
Of course posts and pages are default post types, so it would be valuable to keep them, but then where should we stop? Should we also provide `category-uncategorized`? 😄
 
Jokes aside, IMHO "variable" templates shouldn't be hardcoded.
If anything, we might want to generate their details dynamically by fetching all the site's post types.

---

On the other hand, the reason for moving `single` into posts is definitely debateable.
Should we treat the "post" term as blog article (`wp_post` post type), or as a generic word to indicate any element of any post types (except pages, because they are somehow special 😄)?
I don't have a good answer, and it's always been a confusing part of WP.

For now, given that the sidebar shows "Pages" and "Posts", I'm more inclined to treat "Post" as "any post type", and nest `single` into it.

Eventually, the hierarchy structure will be a design decision, and luckily the structure is flexible enough to make it very easy to move things around. 🙂

Reply:
Gotcha! Thanks for clarifying!